examples/acs2/maze/utils.py

```diff
@@ -34,3 +34,21 @@ def maze_knowledge(population, environment) -> float:
             nr_correct += 1
 
     return nr_correct / len(transitions) * 100.0
+
+
+def print_detailed_knowledge(maze, population):
```

Comment:
A short method doc might be useful here.

Reply:
Done.